pkg/network/server.go

```diff
@@ -253,6 +266,14 @@ func (s *Server) handleInvCmd(p Peer, inv *payload.Inventory) error {
 	return p.WriteMsg(NewMessage(s.Net, CMDGetData, payload))
 }
 
+// handleAddrCmd will process received addresses.
+func (s *Server) handleAddrCmd(p Peer, addrs *payload.AddressList) error {
```

Comment:
is that an implementation of some interface and that's why it returns error?
since right now it always returns nil for some reason

Reply:
I made it return error just to unify handlers even though they don't have any standardized interfaces.